SamSawanJava/src/main/java/com/mycompany/romannumeralconverter/app/Converter.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.mycompany.romannumeralconverter.app;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author ssawan
+ */
+public class Converter {
+
+    //hey it's the api
+    public int converter(String numerals) {
+        return translateNumeralsToNumberValues(numerals.toCharArray());
+    }
+
+    private int translateNumeralsToNumberValues(char[] numerals) {
+        //hate the fact that I am using an arraylist...
+        List numberPair = new ArrayList<>();
+        int sum = 0;
+        for (int i = 0; i < numerals.length; i++) {
+            switch (numerals[i]) {
+                case 'I':
```

Comment:
@ssawan623 This switch statement could be refactored to help eliminate the duplication.  The pattern within each case statement is highlighting this fact.
Other than that, looks good!  :metal: